MAGEDOC-3609-improvements-for-email-templates (#155)
magento/merchdocs: src/marketing/variables-custom.md

  - /system/variables-custom.html
---

> If you know a little basic HTML, you can create custom variables and use a [markup tag]({% link marketing/markup-tags.md -%}) to incorporate them into [email]({% link marketing/email-templates.md -%}) and [newsletter]({% link marketing/newsletter-template.md -%}) templates, and other types of [content]({% link cms/content-elements.md -%}).

Reviewer: ```suggestion
If you know a little basic HTML, you can create custom variables and use a [markup tag]({% link marketing/markup-tags.md -%}) to incorporate them into [email]({% link marketing/email-templates.md -%}) and [newsletter]({% link marketing/newsletter-template.md -%}) templates, as well as other types of [content]({% link cms/content-elements.md -%}).
```

1. Click <span class="btn">Add New Variable</span>.

> 1. Enter an identifier in the **Variable Code** field, using all lowercase characters without spaces. However, the underscore character can be used to represent a space. For example: `my_custom_variable`

Reviewer: ```suggestion
1. Enter an identifier in the **Variable Code** field, using all lowercase characters without spaces. 

    If needed, you can use an underscore character to represent a space. For example: `my_custom_variable`
```

 
1. Enter a **Variable Name**, which is used for internal reference. For example: `My Custom Variable`
   
1. To enter the value that is associated with the variable, do one of the following:

   - In the **Variable HTML Value** field, enter the variable value formatted with simple HTML tags. For example:

>    `<b>This formatted content appears in place of the variable.</b>`

Reviewer: ```suggestion
   `**This formatted content appears in place of the variable.**`
```

Author: Magento doesn't accept Markdown, so the example needs to show how to use HTML tags to format an attribute value.  The <b></b> tags should be visible, but the example should not be formatted, because the taqs aren't applied at this point.

    _New Custom Variable_

1. When complete, click <span class="btn">Save</span>.
   

>   Your custom variable is now available to be inserted into email and newsletter templates and other types of content.  The code that is inserted looks something like the following [markup tag]({% link marketing/markup-tags.md -%}):

Reviewer: ```suggestion
  Your custom variable is now available to be inserted into email and newsletter templates and other types of content.  The code that is inserted looks similar to the following [markup tag]({% link marketing/markup-tags.md -%}):
```
